### PR TITLE
Refactor wideband

### PR DESCRIPTION
**Pull request by SebastianSemper**
Related issue is #66.

### CODE REVIEW

File: eadf/core.py [hunk 1]
```diff
@@ -563,95 +308,26 @@ def evaluatePatternNarrowBand(
 
         def funCoEle(arrCoEle):
             # equation (7) in the EADF Paper
-            return xp.exp(1j * xp.outer(muCoEle, arrCoEle))
+            return xp.exp(xp.outer(1j * muCoEle, arrCoEle))
```
Comment (MichaelDoebereiner) [inline eadf/core.py:310]: Does this change anything?

**Comment by SebastianSemper** [comment]
It's faster, since we only have to do a scalar * vector product instead of
a scalar * matrix product.

**Review by MichaelDoebereiner — CHANGES_REQUESTED** [review]
Nice refactoring! Saves quite a few lines of code.
I have some remarks for the demos:
- The operation.py demo is not running without errors.
- The performance.py is running, but not as expected, since for me the normal EADF is always faster than the performance EADF.
- Whats the plan for the demos in the legacy folder? Are they abandoned and will be removed in the future?

**Review by MichaelDoebereiner — APPROVED** [review]
Now, the operation.py demo works fine for me. The performance problem seems to be related to my personal setup.
So from my side, we're ready to go!

**Comment by SebastianSemper** [comment]
Let's see what @StellarSankalp has to say. :)

**Comment by pawarsp** [comment]
Refactored code works pretty smoothly. As Michael, PerformanceEADF seems to perform slower than normal EADF as shown by the results of performance.py demo. 

Good to go from me as well! May the force be with you ^^

**Review by pawarsp — APPROVED** [review]

**Pull request merged**